### DISCUSSION TITLE
Prompt is not listening, strength settings

### DISCUSSION THREAD

**Discussion by juliux32** [lllyasviel/Fooocus#1486] [Q&A] [answered]
So I was playing around and noticed that my Fooocus is body-shaming girls if I could say so 😅 I tried several namings for a "big" girl, but my images always end with a "regular" size person... 

For example, I put a simple prompt "fat girl on beach, yellow bikini" and during the creation process, on the preview I see that a bigger size person is being created, BUT at around steps 11-14 it suddenly starts to make a person skinnier and with around steps 15-20 it becomes a  "regular" size person...

So my question is... Do I need to change something or write something in a prompt to force Fooocus to use certain prompts no matter what? I saw that in other software you use brackets, numbers, prompt order and etc. to prioritize one over another, but nothing seems to work for me... or I simply doing something wrong

**Comment by juliux32** [comment]
**UPDATE!** 
The same thing is happening with age... I tried to enter the simple prompt "55 year old woman sitting on the bench, short blonde hair" and first 11-14 steps you can clearly see that it is an older woman, BUT then again, generation stops for a moment and it changes to young woman 😕

**Reply by RTalive** [reply]
I have the same situation with postures and facial expressions. Exactly how you describe it.

**Comment by ItsNoted** [comment, marked as answer]
This really depends on what model and settings you use. Try using default Juggernaut model with no style settings.

**Reply by billymambo** [reply]
> This really depends on what model and settings you use. Try using default Juggernaut model with no style settings.

You're the best. Can't believe it was so simple.

**Comment by Xposure345** [comment]
I usually do prompts with nightvisionXL model and no style to see what I get, and usually stay away from the Fooocus Styles.
